arangod/Aql/types.h

```diff
@@ -109,4 +106,20 @@ enum class ExplainRegisterPlan {
 
 }  // namespace arangodb
 
+namespace std {
```

Comment:
Probably these should be moved into RegisterId.h, too. But we can do this later.